web-common/src/features/dashboards/time-series/multiple-dimension-queries.ts

```diff
@@ -58,6 +57,61 @@ interface DimensionTopList {
   totals?: number[];
 }
 
+// TODO: Deprecate this in favor of aggregation queries with a time spine
+function createMetricsViewTimeSeries(
+  ctx: StateManagers,
+  measures: string[],
+  isComparison = false,
+): CreateQueryResult<V1MetricsViewTimeSeriesResponse, HTTPError> {
+  return derived(
+    [
+      ctx.runtime,
+      ctx.metricsViewName,
+      ctx.dashboardStore,
+      useTimeControlStore(ctx),
+    ],
+    ([runtime, metricsViewName, dashboardStore, timeControls], set) => {
+      return createQueryServiceMetricsViewTimeSeries(
```

Comment:
Given this is a new function, mind updating this to use the stable query observer pattern?